api/test/pact/pact-tests/get-for-location-caseworker.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { Pact } from '@pact-foundation/pact'
+import { assert } from 'chai'
+import * as getPort from 'get-port'
+import * as path from 'path'
+import { EnhancedRequest } from '../../../lib/models'
+import { handleCaseWorkerForLocation } from '../../../workAllocation/caseWorkerService'
+
+describe("Work Allocation for location Caseworker API", () => {
+
+    let MOCK_SERVER_PORT
```

Comment:
Not sure Capitals should be used here, as this is not a static constant.

Reply:
This is the normal convention for all the PACT tests. Happy to change it. but that means 32 changes in 8 files. Are you sure about this?

Reply:
Let's change it as we go :) It's a development convention that static constants are in full capitals. But nothing else should be.